alpha/lib/model/LiveEntry.php

```diff
@@ -609,19 +613,28 @@ private function storeInCache($key)
 			KalturaLog::debug("cacheStore is null. cacheType: $cacheType . returning false");
 			return false;
 		}
-		return $cacheStore->set($key, true, kConf::get('media_server_cache_expiry', 'local', self::DEFAULT_CACHE_EXPIRY));
+		KalturaLog::debug("Set cache key [$key] from store [$cacheType] ");
+		$returnValue =  $cacheStore->set($key, true, kConf::get('media_server_cache_expiry', 'local', self::DEFAULT_CACHE_EXPIRY));
+		$ans = $cacheStore->get($key);
+		KalturaLog::debug("Get cache key [$key] from store [$cacheType] returned [$ans] and reurn value was [$returnValue]");
 	}
 	
-	public function setMediaServer($index, $hostname, $applicationName = null)
+	public function setMediaServer($index, $hostname)
```

Comment:
Think about moving the creating of the entryServerNode object to this function and adding status param to this function